vclib/src/commonMain/kotlin/at/asitplus/wallet/lib/agent/HolderAgent.kt

```diff
@@ -69,46 +68,49 @@ class HolderAgent(
      *
      * Note: Revocation credentials should not be stored, but set with [setRevocationList].
      */
-    override suspend fun storeCredential(credential: Holder.StoreCredentialInput)
-            : KmmResult<Holder.StoredCredential> = catching {
-        when (credential) {
-            is Holder.StoreCredentialInput.Vc -> {
-                val vc = validator.verifyVcJws(credential.vcJws, keyPair.publicKey)
-                if (vc !is Verifier.VerifyCredentialResult.SuccessJwt) {
-                    throw VerificationError(vc.toString())
+    override suspend fun storeCredential(credential: Holder.StoreCredentialInput): KmmResult<Holder.StoredCredential> =
+        catching {
```

Comment:
just a minor remark: Maybe you could remove the explicit result type, and move `catching` one line up (to the `fun` definition), to reduce the nesting for the body

Reply:
fixed in f2a300dabe9855916935f35daf1408b409f71cc8